cluster.go

```diff
@@ -61,6 +61,20 @@ func (c *ClusterClient) Close() error {
 	return nil
 }
 
+// PoolLen returns the number of active connections in the pool, for instrumentation
```

Comment:
I would like measure and periodically report the actual number of used pool connections